modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -124,7 +124,8 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             as QCLastChangeTime FROM files
             LEFT JOIN files_qcstatus USING (FileID)
             WHERE OutputType='native' AND AcquisitionProtocolID 
-            NOT IN (1, 2, 3, 52) GROUP BY files.SessionID) nd 
+            IN (".
+            implode(",", array_keys($scan_id_types)) .") GROUP BY files.SessionID) nd
```

Comment:
I don't know where $scan_id_types comes from, but this doesn't seem safe.

It looks like it's all scan id types that exist, making this in clause appear pointless.

Given that AcquisitionProtocolID 1, 2, and 3 don't exist because of the stupid default schema, and 52 in the default schema is 'scout' (which should probably be excluded at the protocol level, not in an imaging browser query), couldn't the same thing by achieved by just taking out the AcquisitionProtocolID part of the where clause?